app/views/spaces/edit.html.erb

```diff
@@ -24,7 +24,7 @@
         </li>
       <% end %>
 
-      <% if space.entrance.furnitures.blank? %>
+      <% if space.entrance_id.present? && space.entrance&.furnitures.blank? %>
```

Comment:
I think I see what you mean Zee when you mentioned https://github.com/zinc-collective/convene/issues/1565#issuecomment-1606518350